src/styles/header.scss

```
@@ -0,0 +1,67 @@
+$navlink-active-color: #00acdc;
+$header-color: #fff;
+$navlink-color: #060b35;
+$navlink-backgroundcolor: #fff;
```

Don't need to duplicate variables with the same values and move variables to the separate file
```suggestion
$navlink-active-color: #00acdc;
$header-color: #fff;
$navlink-color: #060b35;
```